nbs/01_TweetId_Extraction.ipynb

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
<p>A question here, is "outputs" in "data" dir? Are we making this dir somewhere? Also shouldn't we have the code to download and extract 7z from gdrive somewhere because then this would be hard to figure out again in 5 months? Or we don't want to keep this code public?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/55/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Good question, we should remove this code from public now itself. </p><p> </p>